src/pages/Home.vue

```diff
@@ -230,6 +231,31 @@ const copySlotText = (slot: number) => {
   }
 }
 
+const draggedInbox = (x: number, y: number, itemIndex: number, podcastId: string) => {
+  const id =
+    <string>// @ts-ignore
+    document.elementFromPoint(x, y)?.attributes["inbox-id"]?.value
+  const podcast =
+    <string>// @ts-ignore
+    document.elementFromPoint(x, y)?.attributes["inbox-podcastId"]?.value
+    if(id && podcast) {
+      const currentArray = shareStore.getInboxByKey(podcastId) as []
+      const reOrderArray = shareStore.getInboxByKey(podcast) as []
+      if (podcastId === podcast) {
+        const data= moveArrayItemToNewIndex(currentArray, itemIndex,parseInt(id));
+        shareStore.reOrderTOInbox(data, props.podcastId, podcast)
+      } else {
+        const vlaueExist = reOrderArray.includes(currentArray[itemIndex])
+        if(!vlaueExist){
+          reOrderArray.splice( parseInt(id)+1, 0, currentArray[itemIndex] );
+          currentArray.splice(itemIndex, 1)
+          shareStore.reOrderTOInbox(reOrderArray, props.podcastId, podcast)
+          shareStore.reOrderTOInbox(currentArray, props.podcastId, podcastId)
```

Comment:
Weird casing. reorderToInbox?

Reply:
I have updated this functionality,
Please refer: https://i.imgur.com/vAc2V79.png

When the user drags the item to another podcast then we need to update two podcast inboxes so I have used the `reOrderTOInbox` function in a casing

Reply:
Please refer -> https://i.imgur.com/9lBoide.png
according to image.
currently, dev inbox data is one dev and one dev2 item.
now I have dragged the dev item into the dev 2 item so we need to remove the dev inbox item and add this item on dev 2 I have used the `reOrderTOInbox` function two times to add and remove an item on the current inbox.

Reply:
this ^ is still true, no need to reorder within the inbox